contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -98,6 +98,7 @@ install() {
 				type mark_hostonly >/dev/null 2>&1 && mark_hostonly @systemdunitdir@/zfs-import-$_item.service
 			fi
 		done
+		inst "${moddir}/zfs-env-bootfs.service" "${systemdsystemunitdir}/zfs-env-bootfs.service"
```

Comment:
Should this be `inst_simple`?

Reply:
I took this from dracut's [man page](http://man7.org/linux/man-pages/man7/dracut.modules.7.html) which only names `inst <src> [<dst>]`. [dracut-init.sh](https://git.kernel.org/pub/scm/boot/dracut/dracut.git/tree/dracut-init.sh#n182) defines both but there is not much difference, at least at this point... both should be OK here